minion_log.conf

```diff
@@ -0,0 +1,10 @@
+log4perl.rootLogger=DEBUG, LOGFILE
```

Comment:
What is this file for?

Reply:
It's the log file configuration for the Minion queue, so that we can turn on different levels of logs for debugging etc.